### PR TITLE
[Merged by Bors] - doc: recommend `lake exe mk_all` in the README

### PR DESCRIPTION
**Pull request by grunweg**
Instead of the `find` incantation.

-----------

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ You may want to subscribe to the `mathlib4` stream
 * If you added a new file, run the following command to update `Mathlib.lean`
 
   ```shell
-  find Mathlib -name "*.lean" | env LC_ALL=C sort | sed 's/\.lean//;s,/,.,g;s/^/import /' > Mathlib.lean
+  lean exe lint_style
```
Comment (grunweg) [inline README.md:77]:
```suggestion
  lean exe mk_all

**Comment by Rida-Hamadani** [comment]
The Zulip link in the description is broken.
Is it a good idea to do this if it is not working on windows yet? Maybe we can replace the `find` incantation with `scripts/mk_all.sh` until it gets fixed?

**Comment by grunweg** [comment]
If I understand correctly, #13854 and #13850 will fix the script on Windows. (Help testing the second PR is welcome; I don't have a windows computer. Check out that branch, run `lake exe mk_all` and report if this works or not.)
Let's mark this PR as depending on them. (And I'll fix the zulip link shortly.)

**Comment by semorrison** [comment]
bors d+

Sorry about causing the merge conflict you're about to have to resolve!

**Comment by grunweg** [comment]
No worries - thanks for reviewing the dependent PR and this one. Since I just bors-ed the other PR, let me approve this one already.
bors r+

**Pull request closed without merging**